rest/endpoints/user_recordings/profile_recordings.yml

```diff
@@ -466,7 +466,68 @@ paths:
                     type: array
                     items:
                       $ref: "../../snippets/Epg/BroadcastSlotEntity.yml#/BroadcastSlotEntity"
-                      
+
+  # ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+  /user_profiles/{user_profile_id}/recorded_broadcast_slots/{broadcast_slot_id}:
+    get:
+      description: |
+        * **Available since:** 2021-2 (BPLAT-14510)
+      summary: Search for recordings of a given user profile Id and Broadcast slot Id
+      tags:
+        - Recordings
+      operationId: searchRecording
+      parameters:
+        - in: path
+          name: user_profile_id
+          required: true
+          description: The identity of the user profile to fetch recording.
+          schema:
+            type: string
+        - in: path
+          name: broadcast_slot_id
+          required: true
+          description: The identity of the broadcast slot to fetch recording.
+          schema:
+            type: string
+      responses:
+        200:
+          description: "Successful response"
+          content:
+            application/json:
+              schema:
+                type: "object"
+                required:
+                  - data
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: "https://raw.githubusercontent.com/booxmedialtd/api-documentation/master/rest/snippets/UserRecordings/RecordingEntity.yml#/RecordingEntity"
+        403:
+          description: Client attempts to fetch the recordings of a un-authenticated user.
+          content:
+            application/json:
+              schema:
```

Comment:
This should be `JsonApiErrorEntity` as the 404 below. Please also add example data